### DISCUSSION TITLE
Option for Red Teaming

### DISCUSSION THREAD

**Discussion by 72SB02** [Pennyw0rth/NetExec#189] [Ideas]
Hi guy's

I just noted that many detection tools pick up NXC as soon as it for example connects to the ADMIN$ share and most likely failed. Especially on DC's. 

My idea is to get an RT mode that reduces the number of requests as well as some other settings that make it more sneaky.

**Comment by Marshall-Hallenbeck** [comment]
This has come up before and we make it clear this is not a tool to be stealthy with - it's a pentest tool to assist people with executing commands on systems in a network.

If you have any specific examples, I'd be happy to consider them to make the tool better, but things like "reducing the number of requests" are up to the user to do.

I _do_ want to add more options to change certain things like filenames, etc, so there's more customization available for users, but in reality, the way nxc executes things is _not stealthy_ and is _easily detectable_.
